stix_shifter/stix_translation/src/modules/aws_cloud_watch_logs/README.md

```diff
@@ -0,0 +1,542 @@
+# AWS CloudWatchLogs UDS Connector
```

Comment:
Remove UDS here. A connector can be used with any security product that passes in the expected connection object.

Reply:
We will remove it

Reply:
Done